client_with_ssl.go

```diff
@@ -117,36 +119,41 @@ func populateCities(client *hazelcast.Client) {
 
 	_, err := client.SQL().Execute(context.Background(), insertQuery)
 	if err != nil {
-		fmt.Print("FAILED. ", err)
+		// don't panic for duplicated keys.
+		fmt.Errorf("FAILED. %s", err)
 	} else {
 		fmt.Print("OK.")
 	}
 
 	fmt.Print("\nPutting a city into 'cities' map...")
-	cityMap, err := client.GetMap(ctx, "cities")
+	cityMap, err := client.GetMap(context.Background(), "cities")
 	if err != nil {
 		panic(err)
 	}
 
+	// Let's also add a city as object.
 	city := City{Country: "Brazil", CityName: "Rio de Janeiro", Population: 13634274}
-	cityMap.Put(ctx, 8, city)
+	_, err = cityMap.Put(context.Background(), int32(8), city)
+	if err != nil {
+		panic(fmt.Errorf("FAILED. %s", err))
```

Comment:
How about something like:
```
panic(fmt.Errorf("putting into cities: %w", err))
```

Reply:
It completes the sentence, like;
(success) -> `Inserting data via SQL...OK.`
(failed)-> `Inserting data via SQL...FAILED. and error message goes on`

Reply:
Sure, fixed at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/14/commits/0f3b228a288c3bb650761a00488fe04c307fd81a